infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/StreamsSnapshotWriter.java

```diff
@@ -242,6 +242,15 @@ public void apply(LogReplicationEntryMsg message) {
         }
         UUID regularStreamId = opaqueEntry.getEntries().keySet().stream().findFirst().get();
 
+
+        // When there is an upgrading and Active's streams to replicate set has streams not present
```

Comment:
When there is an upgrading -> when an upgrade has been performed and active's ...